### PR TITLE
[Merged by Bors] - use first applied block in epoch for new miner active set

### LINKED ISSUE #4912: miner.ActiveSetFromEpochFirstBlock will use different block depending on synced certificate
if node wasn't online and synced certificates only for several last layers it will use different block for active set compared with 
nodes that were online for all time and have the earliest certificates.

i think that function should ignore certificates and use any applied block regardless of the reason

### PR DESCRIPTION
**Pull request by countvonzero**
## Motivation
Closes #4912 

## Changes
- for new miner use the first applied block for ref ballot active set

### CODE REVIEW

File: miner/util.go [hunk 1]
```diff
@@ -10,14 +10,30 @@ import (
...
+func ActiveSetFromEpochFirstCertifiedBlock(db sql.Executor, epoch types.EpochID) ([]types.ATXID, error) {
 	bid, err := certificates.FirstInEpoch(db, epoch)
 	if err != nil {
 		return nil, err
 	}
+	return activeSetFromBlock(db, bid)
+}
+
+func ActiveSetFromEpochFirstBlock(db sql.Executor, epoch types.EpochID) ([]types.ATXID, error) {
+	result, err := ActiveSetFromEpochFirstCertifiedBlock(db, epoch)
```
Comment (dshulyak) [inline miner/util.go]: this would be a different  block for synced node, i think it should be always first applied regardless of certificate

**Review by dshulyak — APPROVED** [review]

**Comment by dshulyak** [comment]
i think it is this bug https://github.com/spacemeshos/go-spacemesh/issues/4912

**Comment by countvonzero** [comment]
> i think it is this bug #4912

not aware of this. thanks!

**Comment by countvonzero** [comment]
bors merge

**Comment by countvonzero** [comment]
bors merge

**Pull request closed without merging**